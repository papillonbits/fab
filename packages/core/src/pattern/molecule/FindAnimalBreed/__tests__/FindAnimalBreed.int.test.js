import { findDogBreed, findCatBreed } from './FindAnimalBreed.int.story'

jest.mock('../../../../store/thunk/setRandomImage', () => ({ setDogRandomImageThunk: () => {}, setCatRandomImageThunk: () => {} }))

describe('<FindAnimalBreed />', () => {
  describe('Render', () => {
    test('must match findDogBreed', () => {
      expect(global.renderToJSON(findDogBreed())).toMatchSnapshot()
    })

    test('must match findCatBreed', () => {
      expect(global.renderToJSON(findCatBreed())).toMatchSnapshot()
    })
  })
})
