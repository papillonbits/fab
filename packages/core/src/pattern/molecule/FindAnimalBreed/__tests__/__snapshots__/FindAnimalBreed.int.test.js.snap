// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FindAnimalBreed /> Render must match findCatBreed 1`] = `
<div>
  <div
    class="breed"
  >
    <h2>
      A random image of 
      beng
       breed
    </h2>
    <h4>
      Type in new breed then press enter or click search button
    </h4>
    <div
      class="breedSearch"
    >
      <input
        class="input"
        placeholder="beng"
      />
      <button
        class="button"
        type="button"
      >
        Search Cat
      </button>
    </div>
    <div
      class="breedShow"
    >
      <div
        class="image"
      >
        <img
          alt="beng"
          src="https://66.media.tumblr.com/tumblr_m4iejdP8Hb1qd477zo1_250.jpg"
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`<FindAnimalBreed /> Render must match findDogBreed 1`] = `
<div>
  <div
    class="breed"
  >
    <h2>
      A random image of 
      collie
       breed
    </h2>
    <h4>
      Type in new breed then press enter or click search button
    </h4>
    <div
      class="breedSearch"
    >
      <input
        class="input"
        placeholder="collie"
      />
      <button
        class="button"
        type="button"
      >
        Search Dog
      </button>
    </div>
    <div
      class="breedShow"
    >
      <div
        class="image"
      >
        <img
          alt="collie"
          src="https://images.dog.ceo/breeds/collie-border/n02106166_1031.jpg"
        />
      </div>
    </div>
  </div>
</div>
`;
